Dev Containers with Depot guide (#15)
depot/docs: content/guides/docker-build.mdx

---
title: Docker build and buildx build
ogTitle: How to use Depot with your existing Docker build and buildx build commands
description: Use Depot with your existing docker build and docker buildx build commands with our depot configure-docker command.
---

# Docker build and buildx build

## How to use Depot with `docker build` and `docker buildx build`

Depot can directly integrate with your existing `docker build` workflows via a one-time configuration command from our `depot` CLI. See [our instructions for installing our CLI](/docs/cli/installation) if you still need to do so.

> With the CLI installed, you can run `configure-docker` to configure your Docker CLI to use Depot by default for builds:

Reviewer: ```suggestion
With the CLI installed, you can run `configure-docker` to configure your Docker CLI to use Depot as the default handler for `docker build`:
```
